src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx

```diff
@@ -76,12 +123,39 @@ export default class CenterPanel extends React.PureComponent<Props, State> {
   }
 
   render() {
-    const { bodyShape, skinColor, eyeColor, hairColor, avatarAnimation, bodyShapeBaseWearables } = this.props
-    const { isShowingAvatarAttributes } = this.state
+    const { bodyShape, skinColor, eyeColor, hairColor, emote, selectedBaseWearables, visibleItems } = this.props
+    const { isShowingAvatarAttributes, isLoading } = this.state
 
     return (
       <div className="CenterPanel">
-        <ViewPort type={PreviewType.WEARABLE} />
```

Comment:
Would you mind removing wearable from the `PreviewType`?

Reply:
I did https://github.com/decentraland/builder/pull/1993/files/61927c7991e2cee7bd387d16d74101bb6610727d#diff-e0563bb9a79514c665cb8464e26c65f63d270b395d6fa35bf2dae20242fa1fa7L39

Reply:
Ohh, I didn't see it, sorry!